test/unity_test.rb

```diff
@@ -0,0 +1,155 @@
+# test/catalog/unity_test.rb
+
+require "test_helper"
+require "webmock/minitest"
+
+class UnityCatalogTest < Minitest::Test
+  def setup
+    @workspace_url = "https://test.workspace.com"
+    @token = "test_token"
```

Comment:
if anything `# frozen_string_literal: true` would be more valuable here, since you've got these instanced variables.

Otherwise you'd have to `"https://test.workspace.com".freeze` em I believe